marathon_acme/clients.py

```diff
@@ -281,3 +324,28 @@ def get_app_tasks(self, app_id):
         """
         return self.get_json_field(
             'tasks', path='/v2/apps%s/tasks' % (app_id,))
+
+    def get_events(self, callbacks):
+        """
+        Attach to Marathon's event stream using Server-Sent Events (SSE).
+
+        :param callbacks:
+            A dict mapping event types to functions that handle the event data
+        """
+        d = self.request('GET', path='/v2/events', headers={
+            'Accept': 'text/event-stream',
+            'Cache-Control': 'no-store'
+        })
+
+        # We know to expect JSON event data from Marathon, so wrap the
+        # callbacks in a step that decodes the JSON.
+        wrapped_callbacks = {}
```

Comment:
I feel like this pattern could be done in a cleaner way.. I'm just not sure how and I kept getting the scoping of the closure wrong every other way I did it :/